src/quo2/components/messages/system_message.cljs

```diff
@@ -55,7 +55,23 @@
 
 (defmulti sm-render :type)
 
-(defmethod sm-render :deleted [{:keys [state action timestamp-str]}]
+(defmethod sm-render :deleted [{:keys [timestamp-str]}]
+  [rn/view {:align-items     :center
+            :justify-content :space-between
+            :flex            1
+            :flex-direction  :row}
+   [rn/view {:align-items    :center
+             :flex-direction :row}
+    [sm-icon {:icon    :main-icons/delete16
+              :color   :danger
+              :opacity 5}]
+    [text/text {:size  :paragraph-2
+                :style {:color        (get-color :text)
+                        :margin-right 5}}
+     (i18n/label :message-deleted)]
+    [sm-timestamp timestamp-str]]])
```

Comment:
`:deleted` is the same as `:deleted-for-me`, except for the label. Maybe they could both call a `render-deleted` fn and pass in label or keyword? It would be nice if there were a value to differentiate between "deleted" and "deleted-for-me" instead and keep using the same defmethod, but I don't see anything (earlier I was using the 'action' to differentiate between them).

Reply:
Done. I removed the `:deleted-for-me` type and add a `:label` option key.